.github/workflow_scripts/lint_check.sh

```diff
@@ -17,9 +17,9 @@ function lint_check_all {
     lint_check multimodal
     lint_check timeseries
     lint_check common
-    lint_check core
+    # lint_check core
```

Comment:
nit: deleting this line would make it cleaner?
unless we need to refer it for the future

Reply:
We want to add the linting back in future: #4426

Reply:
linting will fail currently, because line-length isn't consistent. But we don't want to refactor it yet because in-progress PRs would go through merge conflict hell.